lib/themes/dosomething/paraneue_dosomething/js/onboarding.js

```diff
@@ -29,16 +29,32 @@ $(document).ready(function() {
   // Enable the Onboarding experiment.
   if (Drupal.settings.dsOnboarding.enabled && localStorage.getItem(localStorageKey) !== 'true') {
     const jsxElementHook = 'jsx-onboarding';
-    const slides = [InstructionSlide]; // ReportbackItemKudosSlide, ContextSlide
+    const slides = [];
```

Comment:
@DFurnes should this be `let`? I wasn't getting any errors. I'm not changing the value from an array to some other type, just adding to the items... so I wasn't sure.